libdnf-cli/progressbar/multi_progress_bar.cpp

```diff
@@ -44,9 +44,6 @@ MultiProgressBar::MultiProgressBar() : total(0, "Total") {
 
 MultiProgressBar::~MultiProgressBar() {
     std::lock_guard<std::mutex> lck(mtx);
```

Comment:
Could you also remove the lock guard from here and `std::mutex mtx;` from MultiProgressBar class?
I think it was meant to prevent some race conditions in creating and deleting progressbars, but now it's probably completely pointless.

Reply:
Thanks, removed.